tests/adapters/test_visa.py

```diff
@@ -101,32 +133,26 @@ def test_write_bytes_read(adapter):
 
 class TestReadBytes:
     @pytest.fixture()
-    def adapter(self, adapter):
+    def adapterR(self, adapter):
```

Comment:
:grin: I was thinking more like `adapter_with_filled_buffer`, but fair enough, it's class-local anyway.